src/csmacnz.Coveralls.Tests/TestFileSystem.cs

```diff
@@ -1,24 +0,0 @@
-using System.Collections.Generic;
-using Beefeater;
-
-namespace csmacnz.Coveralls.Tests
-{
-    public class TestFileSystem : IFileSystem
```

Comment:
I'm not happy you deleted this in place for NSubstitute. Can you roll it back please? and remove the NSubstitute dependency?